src/main/java/com/example/WatchNext/WatchNextApplication.java

```diff
@@ -28,6 +25,6 @@ public static void main(String[] args) {
 	@Override
 	public void run(String... args) throws Exception {
 		roleService.saveDefaultRoles();
-		usersService.saveDefaultUser();
+		//usersService.saveDefaultUser();
```

Comment:
why is this line commented? Never leave commented code without saying why you left it there.
ex: add a TODO: uncomment this line when X is working or implemented